test_gpios.py

```diff
@@ -62,14 +62,15 @@ async def test_gpios(self, value):
     async def test_interrupts(self, pwm):
         FREQUENCY = 50 # Hertz
         DURATION = 2 # seconds
-        ERROR_FACTOR = 0.05
 
         if pwm == "hw":
             pwm_pin = self.hw_pwm_pin
             interrupt = self.hw_interrupt
+            ERROR_FACTOR = 0.05
         else:
             pwm_pin = self.sw_pwm_pin
             interrupt = self.sw_interrupt
+            ERROR_FACTOR = 0.07
```

Comment:
Maybe this is too nitpicky, but now that it's not really a constant any more, I'd rename to lower case. In `vim` you can do this with `gu<movement>`, though I don't know how to do it in other editors.